### ISSUE TITLE
Unable to reference a term in the default section !? :thinking: 

### ISSUE THREAD

**Issue by silopolis** [realtimeprojects/mkdocs-ezglossary#4]
Re,

It most probably must be me, but I'm unable to reference a term defined in the default section:

The term is defined as

```
myterm
:   Myterm tests default glossary section
```

And referenced as:

```
Referencing <:myterm> <_:myterm> in default section.
```

Which renders as "<:myterm>" and "<_:myterm>"

Thanks in advance for your help

**Comment by realtimeprojects** [comment]
Thanks for your feedback.

On the first glance, this seems to be a bug, I try to address this ASAP.

**Comment by realtimeprojects** [comment]
This was in fact a bug since the suggested link syntax lead to some quoting issues.

So the [link syntax has now changed slightly](https://realtimeprojects.github.io/mkdocs-ezglossary/usage/default/#linking) to either

```python
<term:>
``` 

or

```python
<default:term>
```
This is released now with version v1.5.6 and available on pypi.org
